django/core/management/commands/makemigrations.py

```diff
@@ -57,9 +57,16 @@ def add_arguments(self, parser):
             '--check', action='store_true', dest='check_changes',
             help='Exit with a non-zero status if model changes are missing migrations.',
         )
+        parser.add_argument(
+            '--scriptable', action='store_true', dest='scriptable',
+            help='Output only created migration filepaths to stdout; divert logging and prompts to stderr.',
+        )
 
     def log(self, msg):
-        self.stdout.write(msg)
+        if self.scriptable:
+            self.stderr.write(msg)
+        else:
+            self.stdout.write(msg)
```

Comment:
I noticed that all logs and prompts have `ERROR` style when using `--scriptable`, e.g.:

![image](https://user-images.githubusercontent.com/2865885/148344507-ada0d115-4a48-4001-81a2-b62c919c5e45.png)
![image](https://user-images.githubusercontent.com/2865885/148344684-e00db0d8-c25f-45fc-ba54-9dfef13eac7c.png)

We could create a copy of `stderr` without the `ERROR` style and use it where appropriate :thinking: 
```diff
diff --git a/django/core/management/commands/makemigrations.py b/django/core/management/commands/makemigrations.py
index cdb200f22e..096702814c 100644
--- a/django/core/management/commands/makemigrations.py
+++ b/django/core/management/commands/makemigrations.py
@@ -6,7 +6,7 @@ from itertools import takewhile
 from django.apps import apps
 from django.conf import settings
 from django.core.management.base import (
-    BaseCommand, CommandError, no_translations,
+    BaseCommand, CommandError, no_translations, OutputWrapper
 )
 from django.db import DEFAULT_DB_ALIAS, OperationalError, connections, router
 from django.db.migrations import Migration
@@ -62,9 +62,17 @@ class Command(BaseCommand):
             help='Output only created migration filepaths to stdout; divert logging and prompts to stderr.',
         )
 
+    def __init__(self, stdout=None, stderr=None, no_color=False, force_color=False):
+        super().__init__(stdout, stderr, no_color, force_color)
+        if no_color:
+            self.stderr_log = self.stderr
+        else:
+            # stderr without the ERROR style.
+            self.stderr_log = OutputWrapper(stderr or sys.stderr)
+
```

Reply:
What if `--scriptable` just implied `--no-color`?

Reply:
Good idea :+1: